salt/datadog/init.sls

```diff
@@ -36,6 +36,7 @@ datadog-agent:
     - installed
     - require:
       - pkgrepo: datadog_repo
+      - mount: {{ swap_path }}
```

Comment:
probably needs to be wrapped with the same if statement as the swap creation itself